.env

```diff
@@ -0,0 +1,3 @@
+PORT=3000
```

Comment:
Dont push the .env file, share it via other ways than pushing it to github, 
add .env file to a new ".gitignore" file so that it does not get pushed with the code